dev/tasks/update-mockgcp

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd ${REPO_ROOT}
+
+cd ${REPO_ROOT}/mockgcp
+
+# Find latest SHAs
+GOOGLEAPIS_SHA=$(git ls-remote --exit-code https://github.com/googleapis/googleapis  refs/heads/master | awk '{print $1}' )
+
+# Update version markers
+cat > git.versions <<EOF
```

Comment:
I recommend that we write the GIT SHA in the Makefile rule, as opposed to in the task script. There could be manual runs of the Makefile, in which case we also need to record the SHA. WDYT?